src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -218,24 +214,21 @@ export const SPECIAL_CURRENCY_INFO: {
     displayBuyCrypto: true,
     displayIoniaRewards: true,
     isImportKeySupported: true,
-    isStakingSupported: true,
-    isPaymentProtocolSupported: true
+    isStakingSupported: true
   },
   bitcoinsv: {
     maxSpendTargets: UTXO_MAX_SPEND_TARGETS,
     initWalletName: lstrings.string_first_bitcoin_sv_wallet_name,
     chainCode: 'BSV',
     keysOnlyMode: true,
-    isImportKeySupported: true,
-    isPaymentProtocolSupported: true
```

Comment:
bitcoinsv is not in the new `PAYMENT_PROTOCOL_MAP`. Not an immediate problem, since we can't send it anyhow, but perhaps we should keep the map up-to-date.